src/reflect/scala/reflect/internal/Names.scala

```diff
@@ -37,8 +37,13 @@ trait Names extends api.Names {
   protected def synchronizeNames: Boolean = false
   private val nameLock: Object = new Object
 
+
   /** Memory to store all names sequentially. */
-  var chrs: Array[Char] = new Array[Char](NAME_SIZE)
+  private[this] var _chrs: Array[Char] = new Array[Char](NAME_SIZE) // TODO this ought to be private
+  @deprecated("Don't access name table contents directly.", "2.12.9")
+  def chrs: Array[Char] = _chrs
+  @deprecated("Don't access name table contents directly.", "2.12.9")
+  def chrs_=(cs: Array[Char]) = _chrs = cs
```

Comment:
Where is this used?

Reply:
I strongly doubt it is ever used anywhere, I was just being conservative to keep the current API in a minor release.